doc/02-Installation.md

```diff
@@ -0,0 +1,125 @@
+<!-- {% if index %} -->
+# Installing Icinga Notifications
+
+The recommended way to install Icinga Notifications is to use prebuilt packages
+for all supported platforms from our official release repository.
+Please follow the steps listed for your target operating system,
+which guide you through setting up the repository and installing Icinga Notifications.
+
+To upgrade an existing Icinga Notifications installation to a newer version,
+see the [Upgrading](04-Upgrading.md) documentation for the necessary steps.
+
+Before installing Icinga Notifications, make sure you have installed [Icinga 2](https://icinga.com/docs/icinga-2)
+with the [`api` feature](https://icinga.com/docs/icinga-2/latest/doc/12-icinga2-api/) configured and enabled.
+The Icinga 2 installation documentation covers all the necessary steps.
+
+<!-- {% else %} -->
+
+## Setting up the Database
+
+A MySQL (≥5.5), MariaDB (≥10.1), or PostgreSQL (≥9.6) database is required to run Icinga Notifications.
+Please follow the steps listed for your target database,
+which guide you through setting up the database and user and importing the schema.
+
+### Setting up a MySQL or MariaDB Database
+
+If you use a version of MySQL < 5.7 or MariaDB < 10.2, the following server options must be set:
+
+```
+innodb_file_format=barracuda
+innodb_file_per_table=1
+innodb_large_prefix=1
+```
+
+Set up a MySQL database for Icinga Notifications:
+
+```
+# mysql -u root -p
+
+CREATE DATABASE icinga-notifications;
+CREATE USER 'icinga-notifications'@'localhost' IDENTIFIED BY 'CHANGEME';
+GRANT ALL ON icinga-notifications.* TO 'icinga-notifications'@'localhost';
```

Comment:
Using `-` in the database and user names requires additional quoting. I'd suggest using `_` instead. Alternatively, other components also use very generic database names in their documentation as well (like `director`, `reporting`, `x509`, yes, as the full database names), so we might also go with just `notifications`.